.azure-pipelines/fluentNetSdk.yml

```diff
@@ -0,0 +1,37 @@
+pr:
+  branches:
+    include:
+    - master
+    - search-preview
```

Comment:
These branches don't look interesting to your repo. I would remove all but master.

Reply:
done